src/classes/TEST_RecurringDonationBuilder.cls

```diff
@@ -579,6 +602,32 @@ public without sharing class TEST_RecurringDonationBuilder {
         return rd;
     }
 
+    /***
+    * @description Retrieves the value of CreatedDate from valuesByFieldName and removes the key
+    * @return String
+    */
+    private String getCreatedDateAndRemoveKey() {
```

Comment:
The method can be removed to just `getCreatedDate()`.

Reply:
I'd like to push back a bit on this.  The method name ought to describe what the method actually does, and in this case, the method does more than just get the created date.